src/blm.jl

```diff
@@ -40,48 +10,127 @@ function postprocess(::MCMC, chain::MCMCChains.Chains, f::FormulaTerm, df::DataF
 end
 
 """
-    blm(f::FormulaTerm, df::DataFrame, inferencestrat::InferenceStrategy=MCMC(NUTS(),1000); alphaprior, betaprior, sigmaprior)
+Internal function that specifies default priors for non-Normal likelihood BLMs. Additional priors for custom model
+definitions may be passed in `kwargs`.
+"""
+blmpriors(
+    ::Type{<:Distribution};
+    α_prior = Normal(),
+    β_prior = (X,y) -> MvNormal(zeros(size(X,2)),1),
+    kwargs...
+) = (α_prior=α_prior,β_prior=β_prior,kwargs...)
```

Comment:
Multiline stuff should be in `function ... end` syntax, usually -- minor thing, but I think it tends to make things more readable.

```suggestion
function blmpriors(
    ::Type{<:Distribution};
    α_prior = Normal(),
    β_prior = (X,y) -> MvNormal(zeros(size(X,2)),1),
    kwargs...
)
    return (α_prior=α_prior,β_prior=β_prior,kwargs...)
end
```